pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -552,6 +539,13 @@ func (r *ReconcileLogStorage) Reconcile(ctx context.Context, request reconcile.R
 			r.status.SetDegraded("Failed to create or update Elasticsearch lifecycle policies", err.Error())
 			return reconcile.Result{}, err
 		}
+
+		if esLicenseType == render.ElasticsearchLicenseTypeBasic {
+			if err = r.checkOIDCUsersEsResource(ctx); err != nil {
```

Comment:
I'm not sure anymore if we need to block LogStorage on the OIDC resources. Is them not being present going to be a problem with LogStorage or just for the manager? Or are they used by Kibana which is launched by LogStorage.
You mentioned that going from Manager UI to Kibana wouldn't work when using SSO, but it isn't even possible to get to Kibana until Manager is available anyway.
But I guess if SSO is changed then LogStorage should be Degraded if there is a known resource missing that Kibana would need.

Assuming we should keep this check, can you add a comment here summarizing the reason why we're checking and degrading LogStorage if these resources don't exist.

Reply:
Yes, right, if those resources are missing SSO to Kibana will be affected, and Kibana is part of LogStorage so it makes sense to degrade it.